src/util/spectra.jl

```diff
@@ -55,20 +55,23 @@ function apply_doppler_boost!(spectra::AS, dict::AbstractDict ) where { AS<:Abst
         doppler_factor *= calc_doppler_factor(dict[:drift_rv])
     end
     =#
-    if !haskey(dict,:ssb_rv) && !haskey(dict,:ssbz) && !have_issued_ssb_warning
+    if !haskey(dict,:ssb_rv) && !haskey(dict,:ssbz) && !haskey(dict,:ssb_rv_kmps) && !have_issued_ssb_warning
```

Comment:
If we're going to allow either of these, then let's add a check that both aren't set.

Reply:
I updated this so a warning is printed if both are set, so the user knows which is used by default.